Onboarding/menu-personalised.md

```diff
@@ -0,0 +1,673 @@
+end
+
+card ManageUpdate, then: ManageUpdates do
+  update_contact(topic: "@selected_topic")
+end
+
+card ManageData, then: DataSettings do
+  update_contact(topic: "@selected_topic")
+end
+
+card TakeATour, then: GoToTakeATour do
+  update_contact(topic: "@selected_topic")
+end
+
+card About, then: AboutPrivacy do
+  update_contact(topic: "@selected_topic")
+end
+
+```
+
+## Health Guide Generic
+
+```stack
+card HealthGuideGeneric, then: DisplayHealthGuideGeneric do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_health_guide_generic"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  menu_items = map(message.list_items, & &1.value)
+end
+
+card DisplayHealthGuideGeneric, then: DisplayHealthGuideGenericError do
+  selected_topic =
+    list("Select option",
+      CheckUp: "@menu_items[0]",
+      TakeQuiz: "@menu_items[1]",
+      Challenge: "@menu_items[2]",
+      Practise: "@menu_items[3]",
+      Generic1: "@menu_items[4]",
+      Generic2: "@menu_items[5]",
+      Generic2: "@menu_items[6]",
+      MainMenu: "@menu_items[7]"
+    ) do
+      text("@message.message")
+    end
+end
+
+card DisplayHealthGuideGenericError, then: DisplayHealthGuideGenericError do
+  selected_topic =
+    list("Select option",
+      CheckUp: "@menu_items[0]",
+      TakeQuiz: "@menu_items[1]",
+      Challenge: "@menu_items[2]",
+      Practise: "@menu_items[3]",
+      Generic1: "@menu_items[4]",
+      Generic2: "@menu_items[5]",
+      Generic3: "@menu_items[6]",
+      MainMenu: "@menu_items[7]"
+    ) do
+      text("@list_error_text")
+    end
+end
+
+card CheckUp, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card TakeQuiz, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Challenge, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Practise, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Generic1, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Generic2, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card Generic3, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+card MainMenu, then: ArticleTopic01Secondary do
+  update_contact(topic: "@selected_topic")
+end
+
+```
+
+```stack
+card DomainShowcase do
+  log("DomainShowcase")
+end
+
+card MainMenu do
+  log("MainMenu")
+end
```

Comment:
I don't think you need this. Anything that uses this should go to `PersonalisedMenu`?